examples/flutter_gallery/lib/demo/material/text_form_field_demo.dart

```diff
@@ -233,7 +233,7 @@ class TextFormFieldDemoState extends State<TextFormFieldDemo> {
                 TextFormField(
                   decoration: const InputDecoration(
                     border: OutlineInputBorder(),
-                    hintText: 'Tell us about yourself',
+                    hintText: 'Tell us about yourself (e.g., write down what you do or what hobbies you have)',
```

Comment:
yes.  It will use the maxLines value from TextField. Here's the screenshot.
![screen shot 2018-12-12 at 10 40 34 am](https://user-images.githubusercontent.com/121966/49843520-7fa57000-fdfa-11e8-97f3-c38c0850210b.png)